app/controllers/interviews_controller.rb

```diff
@@ -0,0 +1,63 @@
+class InterviewsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :check_interviewer
+  add_breadcrumb "interviews", :interviews_path
+  def index
+    @interviews = Interview.paginate(page: params[:page], :per_page => 5)
+  end
+	
+  def new
+    @interview = Interview.new
+    add_breadcrumb "new_interview", new_interview_path
+  end
+
+  def show
+    @interview = Interview.find(params[:id])
+    add_breadcrumb @interview.id, interview_path(@interview)
+    rescue ActiveRecord::RecordNotFound
+      flash[:danger] = 'Interview does not exist!'
+      redirect_to interviews_path
+  end
+
+  def create
+    @interview = Interview.new(interview_params)
+      if @interview.save
+        redirect_to passage_interview_path(@interview), notice: 'Interview was successfully created'
+      else
+        render :new
+      end
+  end
+
+  def destroy
+    Interview.find(params[:id]).destroy
+    flash[:success] = 'Interview deleted'
+    redirect_to :back
+  end
+
+  def passage
+    @interview = Interview.find(params[:id])
+    add_breadcrumb 'passage_interview_'+(@interview.id).to_s, passage_interview_path(@interview)
+    @id_questions = []
+    @questions = @interview.template.questions
+    @all_questions = Question.all
+    @all_questions.each do |aq|
+      @questions.each do |q|
+        @id_questions.push(aq.id)
+        InterviewQuestion.create(question_id: aq.id, interview_id: @interview.id) if aq.content == q.content
+      end 
```

Comment:
In this case we iterate through all questions in DB and its bad for performance. 
Maybe try to  you use @tempalte.questions instead?